doc/index.rst

```diff
@@ -16,9 +16,9 @@ Highlights
 **Dipy 0.10** is now available. New features in this release include: 
 
 
-- FVTK 2.0: a new visualization framework. For example, see: 
 - DKI: Thanks to support through the Google Summer of Code, to fund work done by Rafael Henriques, we now have the only open-source implementation of DKI.
```

Comment:
Is is really the only open source DKI implementation? A quick google search gives another one on github on the first link. Might be the only one in python though.